### PR TITLE
Add MiniVox (ACML 2021) and VoiceID (Interspeech 2020)

### PR DESCRIPTION
**Pull request by doerlbh**
Hi wq2012, in this pull request, I made three proposed changes:

1. Add two references:
MiniVox (ACML 2021) and VoiceID (Interspeech 2020)

2. Create special topics "online speaker diarization" as it's a pretty cool special scenario which is quite different from most of the diarization systems. In addition to the two new works above, I included two other existing work in this readme (the transformer work and the relational network work). 

3. Since MiniVox has a GitHub repo that host both a dataset and the codes to evaluate them, they are also referenced in the framework and software sections, as instructed.

Thank you for a great repo and sharing these resources! Have a great day!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,19 +74,24 @@ Team in the Inaugural DIHARD Challenge](https://www.isca-speech.org/archive/Inte
 * [ODESSA at Albayzin Speaker Diarization Challenge 2018](https://www.isca-speech.org/archive/IberSPEECH_2018/pdfs/IberS18_AE-5_Patino.pdf), 2018
 * [Joint Discriminative Embedding Learning, Speech Activity and Overlap Detection for the DIHARD Challenge](https://www.isca-speech.org/archive/Interspeech_2018/pdfs/2304.pdf), 2018
 
+
+
 ### Other
 
 #### 2021
 
+* [Speaker Diarization as a Fully Online Bandit Learning Problem in MiniVox](https://proceedings.mlr.press/v157/lin21c/lin21c.pdf)
```
Comment (wq2012) [inline README.md]: These papers are added more than once. Can you make sure each paper is only added once?

File: README.md [hunk 2]
```diff
@@ -254,6 +267,7 @@ Team in the Inaugural DIHARD Challenge](https://www.isca-speech.org/archive/Inte
 | [Fisher English Training Speech Part 1 Speech](https://catalog.ldc.upenn.edu/LDC2004S13) | [Fisher English Training Speech Part 1 Transcripts](https://catalog.ldc.upenn.edu/LDC2004T19)| en | $7000.00 + $1000.00 |
 | [Fisher English Training Part 2, Speech](https://catalog.ldc.upenn.edu/LDC2005S13) | [Fisher English Training Part 2, Transcripts](https://catalog.ldc.upenn.edu/LDC2005T19) | en | $7000.00 + $1000.00 |
 | [VoxConverse](https://github.com/joonson/voxconverse) | TBD | TBD | Free | VoxConverse is an audio-visual diarisation dataset consisting of over 50 hours of multispeaker clips of human speech, extracted from YouTube videos |
+| [MiniVox Benchmark](https://github.com/doerlbh/MiniVox) | [MiniVox Benchmark](https://github.com/doerlbh/MiniVox) | Fen | Free | MiniVox is an automatic framework to transform any speaker-labelled dataset into continuous speech datastream with episodically revealed label feedbacks. |
```
Comment (wq2012) [inline README.md]: What is "Fen"?

File: README.md [hunk 3]
```diff
@@ -59,6 +59,13 @@ To add items to this page, simply send a pull request. ([contributing guide](CON
 * [Joint Speech Recognition and Speaker Diarization via Sequence Transduction](https://arxiv.org/abs/1907.05337), 2019
 * [Says who? Deep learning models for joint speech recognition, segmentation and diarization](https://ieeexplore.ieee.org/abstract/document/8462375), 2018
 
+#### Online speaker diarization
+
+* [Turn-to-Diarize: Online Speaker Diarization Constrained by Transformer Transducer Speaker Turn Detection](https://arxiv.org/abs/2109.11641), 2021
```
Comment (wq2012) [inline README.md]: Make sure each paper only appear once on this paper.

Comment (doerlbh) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (doerlbh) [reply README.md]: Typo. Fixed.

File: README.md (hunk 1 above)
Comment (doerlbh) [reply README.md]: Fixed. Thanks for the reminder!

**Pull request merged**